Source/Core/Core/HW/EXI/EXI_DeviceEthernetBase.h

```diff
@@ -194,13 +185,11 @@ enum RecvStatus
   DESC_RERR = 0x80
 };
 
-#define BBA_RECV_SIZE 0x800
-
-class CEXIETHERNET : public IEXIDevice
+class CEXIEthernetBase : public IEXIDevice
 {
 public:
-  CEXIETHERNET();
-  virtual ~CEXIETHERNET();
```

Comment:
I wonder if we should leave the `Deactivate` method alive (for symmetry with `Activate`) instead of moving it into the dtor. Any thoughts on that?

Reply:
Activate is being called sometime during runtime (from MXCommandHandler). But Deactivate() was only called from inside the destructor.
We could instead just activate from the beginning?

Reply:
That would also be fine, assuming it would be symmetric (since it is a lot easier to find that way). I'm not familiar enough with that part of the code to know whether it can (or should) be in the ctor though.

Reply:
Right now with this Activate() approach we would skip connecting to TAP or server for games which don't use the LAN adapter (like luigi's mansion for example).